### ISSUE TITLE
Play a sound when location service is turned off.

### ISSUE THREAD

**Issue by nikhilnayak98** [tananaev/traccar-client-android#236]
How to play a sound when location service is turned off.

**Comment by tananaev** [comment]
This feature makes very little sense to me. Please explain the use case.

**Comment by nikhilnayak98** [comment]
If you are tracking a mobile and that user turns off the gps or mobile data then an alert sound is played.

**Comment by tananaev** [comment]
Do you want to play it on the mobile phone?

**Comment by nikhilnayak98** [comment]
Yes on the mobile phone

**Comment by tananaev** [comment]
Use case still doesn't make sense. I'm assuming that if user disables location, he's doing it consciously.

**Comment by nikhilnayak98** [comment]
Yes the alert sound is to prevent the user from disabling gps. For example
lets say an employee is tracked by their company but if he turns off the
gps then the alert sound will play until he turns it back on.

**Comment by tananaev** [comment]
Let's see if anybody else is interested. It doesn't sound like a very useful feature.

**Comment by sangzier** [comment]
Sounds a good idea for me

**Comment by nikhilnayak98** [comment]
It is a very useful feature from a business point of view. If your employee is turning off the gps and going away from the route he is assigned to then at that time it is a useful feature. The alert siren will not stop until the gps is turned back on.

**Comment by luke-jr** [comment]
Why not just call your employee when the expected report doesn't come in, and complain?
